apps/web/composables/useCategoriesSearch/useCategoriesSearch.ts

```diff
@@ -24,76 +46,44 @@ export const useCategoriesSearch: UseCategoriesSearchMethodsReturn = () => {
     lastOnPage: 0,
     itemsPerPage: 30,
   });
-  const fetchContentCategories = async () => {
-    if (state.value.loadingContent || !state.value.hasMoreContent) return;
-
-    state.value.loadingContent = true;
-
-    try {
-      const { data } = await useAsyncData(() =>
-        useSdk().plentysystems.getCategoriesSearch({
-          level: 1,
-          type: 'content',
-          page: state.value.contentPage,
-          itemsPerPage: 30,
-          sortBy: 'position_asc',
-          with: 'details,clients',
-        }),
-      );
 
-      const result: CategoryData = data.value?.data ?? createEmptyCategoryData();
-      state.value.contentItems.push(...result.entries);
-      state.value.hasMoreContent = !result.isLastPage;
-      state.value.contentPage++;
-    } catch (error) {
-      console.error('Error fetching content categories:', error);
-    } finally {
-      state.value.loadingContent = false;
-    }
-  };
+  const fetchCategories = async (categoryType: 'item' | 'content') => {
```

Comment:
```suggestion
  const fetchCategories = async (categoryType: 'item' | 'content' | 'item, content') => {
```
We should probably create a type for this.

Reply:
We'll refactor separately.